packages/idos-sdk-js/src/lib/grants/grant.ts

```diff
@@ -1,10 +1,12 @@
 export default class Grant {
-  ownerAddress: string;
+  ownerHumanId?: string;
```

Comment:
```suggestion
  ownerUserId?: string;
```

And all other occurences, too.